pkgs/tools/graphics/epub-thumbnailer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epub-thumbnailer";
+  version = "unstable-2022-04-22";
+
+  src = fetchFromGitHub {
+    owner = "marianosimone";
+    repo = pname;
+    rev = version;
+    sha256 = "0mv0zd74g4xfi2pvl8k0z6qvf1987lwk47ia8cyr51capk95p4rp";
+  };
+
+  # The installer is a simple python script, so no tests nor build steps are necessary
+  dontBuild = true;
+  doCheck = false;
```

Comment:
Unless this actually fixes the build you should just leave these alone

Reply:
Please also at least add a `pythonImportsCheck` to have at least some kinda of testing.

Reply:
Thanks for the review guys! Unless I set those options to the way they are, the build fails, I can get the error log if necessary.
@thiagokokada  does using pythonImportsChecks inside buildPythonApplications allowed? I gonna test it but since it isn't a module per se I don't see how it could be imported.

Reply:
In this case, at least a `installCheckPhase` or a `testVersion` to make sure that the script is packaged correctly would be good enough.

Reply:
When you asked about the installCheckPhase I imagined that I'd try to use something like `epub-thumbnailer --help` but the script in question does not support it, nor the '-h' flag. So I thought about using the script with some epub file but that would require a reliable url for an epub that would not break, so I'm not sure where to find one.
It does support an url as an input but it would make the derivation impure. With that in mind I could use fetchurl to get the file, but it does not solve the problem with url reliability.

I was thinking that maybe I should just check that the script file exists in the `$out/bin/` folder in the installCheckPhase.